src/server/handlers.rs

```diff
@@ -0,0 +1,24 @@
+use actix_web::{HttpRequest, HttpResponse, Responder};
+
+pub async fn get_handler(_req: HttpRequest) -> impl Responder {
+    HttpResponse::Ok()
+        .content_type("application/json")
+        .body("{}")
+}
+#[actix_rt::test]
+async fn test_index() {
+    use super::*;
+    use actix_web::test;
+    let mut app = test::init_service(
+                        App::new().service(
+                        web::resource("/1.0/sync/1.5")
+                        .route(web::post().to(get_handler)))
+                    ).await;
+
+    let req = test::TestRequest::post()
+                                    .uri("/1.0/sync/1.5")
+                                    .to_request();
+    let res = test::call_service(&mut app, req).await;
+
+    assert_eq!(res.status(), 200, "/1.0/sync/1.5 should return 200");
+}
```

Comment:
Probably should also include a few negative tests (e.g. invalid entries that should not match the regular expression) to make sure that things don't slip through that shouldn't.

Reply:
I didn't get the clear picture. Can you give me some examples of how should I add negative test?

Reply:
A negative test is one that should fail for an expected reason. In this case it might be worth creating values that contains invalid characters like 
`12345678|1234-1234-1234-90abcdef` 
`12345678~90ab-cdef-1234-567890ab` 

or one that is an incorrect length like
`12345678-1234-1234-1234-1234567890abcdef-1234`